gemc_api_geometry.py

```diff
@@ -185,6 +175,8 @@ def publish(self, configuration):
 
 				dn.write(lstr)
 
+	# @mariakzurek: in polycone the zplane and radious order are swapped w.r.t. gmc2 implementation
+	# is that how it should be?  
```

Comment:
@maureeungaro Please check this comment

Reply:
Yes this is wanted. This is so that in gemc3 the parameters order matches exactly the geant4 constructor order

Reply:
Ok, great!